{
    "active_lock_reason": null,
    "assignee": {
        "avatar_url": "https://avatars.githubusercontent.com/u/1359421?v=4",
        "events_url": "https://api.github.com/users/justinmk/events{/privacy}",
        "followers_url": "https://api.github.com/users/justinmk/followers",
        "following_url": "https://api.github.com/users/justinmk/following{/other_user}",
        "gists_url": "https://api.github.com/users/justinmk/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/justinmk",
        "id": 1359421,
        "login": "justinmk",
        "node_id": "MDQ6VXNlcjEzNTk0MjE=",
        "organizations_url": "https://api.github.com/users/justinmk/orgs",
        "received_events_url": "https://api.github.com/users/justinmk/received_events",
        "repos_url": "https://api.github.com/users/justinmk/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/justinmk/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/justinmk/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/justinmk",
        "user_view_type": "public"
    },
    "assignees": [
        {
            "avatar_url": "https://avatars.githubusercontent.com/u/1359421?v=4",
            "events_url": "https://api.github.com/users/justinmk/events{/privacy}",
            "followers_url": "https://api.github.com/users/justinmk/followers",
            "following_url": "https://api.github.com/users/justinmk/following{/other_user}",
            "gists_url": "https://api.github.com/users/justinmk/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/justinmk",
            "id": 1359421,
            "login": "justinmk",
            "node_id": "MDQ6VXNlcjEzNTk0MjE=",
            "organizations_url": "https://api.github.com/users/justinmk/orgs",
            "received_events_url": "https://api.github.com/users/justinmk/received_events",
            "repos_url": "https://api.github.com/users/justinmk/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/justinmk/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/justinmk/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/justinmk",
            "user_view_type": "public"
        }
    ],
    "author_association": "NONE",
    "body": "I've found similar issues but not quite the same I have (and also no solution) so I decided to post it.\r\n\r\n - `nvim --version`\r\n```\r\nNVIM v0.2.0\r\nBuild type: Release\r\nCompilation: /usr/local/Homebrew/Library/Homebrew/shims/super/clang -Wconversion -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_MSGPACK_HAS_FLOAT32 -DNDEBUG -DDISABLE_LOG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -I/tmp/neovim-20170707-80027-1bg0wcx/neovim-0.2.0/build/config -I/tmp/neovim-20170707-80027-1bg0wcx/neovim-0.2.0/src -I/usr/local/include -I/usr/local/include -I/usr/local/include -I/usr/local/include -I/usr/local/include -I/usr/local/include -I/usr/local/opt/gettext/include -I/usr/include -I/tmp/neovim-20170707-80027-1bg0wcx/neovim-0.2.0/build/src/nvim/auto -I/tmp/neovim-20170707-80027-1bg0wcx/neovim-0.2.0/build/include\r\nCompiled by brew@Sierra-2.local\r\n\r\nOptional features included (+) or not (-): +acl   +iconv    +jemalloc +tui      \r\nFor differences from Vim, see :help vim-differences\r\n\r\n   system vimrc file: \"$VIM/sysinit.vim\"\r\n  fall-back for $VIM: \"/usr/local/Cellar/neovim/0.2.0_1/share/nvim\"\r\n\r\n```\r\n- Operating system/version: **macOS Sierra 10.12.5**\r\n- Terminal name/version: **iTerm Build 3.1.20170619-nightly**\r\n- `$TERM`: **xterm-256color**\r\n- `pip2 show neovim`: \r\n```\r\nName: neovim\r\nVersion: 0.1.13\r\nSummary: Python client to neovim\r\nHome-page: http://github.com/neovim/python-client\r\nAuthor: Thiago de Arruda\r\nAuthor-email: tpadilha84@gmail.com\r\nLicense: Apache\r\nLocation: /usr/local/lib/python2.7/site-packages\r\nRequires: msgpack-python, trollius, greenlet\r\n```\r\n- `pip3 show neovim`:\r\n```\r\nName: neovim\r\nVersion: 0.1.13\r\nSummary: Python client to neovim\r\nHome-page: http://github.com/neovim/python-client\r\nAuthor: Thiago de Arruda\r\nAuthor-email: tpadilha84@gmail.com\r\nLicense: Apache\r\nLocation: /usr/local/lib/python3.6/site-packages\r\nRequires: msgpack-python, greenlet\r\n```\r\n\r\n`:CheckHealth` reports nothing out of the ordinary\r\n\r\n### Actual behaviour\r\n\r\nEvery time I open Neovim it outputs the following message. After that, everything works just fine.\r\n\r\n```\r\nfunction <SNR>54_InitializePythonBuiltin[23]..provider#python#Call[9]..remote#host#Require[10]..provider#pythonx#Require, line 15\r\nVim(if):E475: Invalid argument: Channel id must be a positive integer\r\nFailed to load python host. You can try to see what happened by starting nvim with $NVIM_PYTHON_LOG_FILE set and opening the generated log file. Also, the host stderr is available in messages.\r\nPress ENTER or type command to continue\r\n```\r\n\r\n### Expected behaviour\r\n\r\nI expect Neovim to open without showing me something is wrong.\r\n\r\n### Things I've tried\r\n\r\nRemoved and reinstalled `python`, `python3`, `neovim`. Installed either `neovim` stable and `neovim --HEAD` on homebrew. The error persists.\r\n\r\nThe error does not occur when I run `nvim -u NONE`\r\n",
    "closed_at": null,
    "closed_by": null,
    "comment_data": [
        {
            "author_association": "NONE",
            "body": "had a similar issue, I reinstalled packages based on those you tried and `python` was missing. I also removed ~/.config/nvim. All is working now.",
            "created_at": "2017-07-28T15:49:46Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-318690010",
            "id": 318690010,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMxODY5MDAxMA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 1,
                "laugh": 0,
                "rocket": 0,
                "total_count": 1,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/318690010/reactions"
            },
            "updated_at": "2017-07-31T17:04:00Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/318690010",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/18217279?v=4",
                "events_url": "https://api.github.com/users/carlosvigil/events{/privacy}",
                "followers_url": "https://api.github.com/users/carlosvigil/followers",
                "following_url": "https://api.github.com/users/carlosvigil/following{/other_user}",
                "gists_url": "https://api.github.com/users/carlosvigil/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/carlosvigil",
                "id": 18217279,
                "login": "carlosvigil",
                "node_id": "MDQ6VXNlcjE4MjE3Mjc5",
                "organizations_url": "https://api.github.com/users/carlosvigil/orgs",
                "received_events_url": "https://api.github.com/users/carlosvigil/received_events",
                "repos_url": "https://api.github.com/users/carlosvigil/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/carlosvigil/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/carlosvigil/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/carlosvigil",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I had the *exact* same issue on a fresh ubuntu 12.04 install.  Unfortunately, I changed a few things I'm not precisely sure of how I got rid of it.\r\n\r\nMy only tip is to try tinkering with adding this to your init.vim: `let g:python3_host_prog = '/home/eric/.pyenv/versions/3.6.2/bin/python3'`.  Please note that using the full unabbreviated path was necessary.  When I replace that path with `~/.pyenv/versions/3.6.2/bin/python3` it triggers the error you're mentioning.",
            "created_at": "2017-07-31T14:51:54Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-319090871",
            "id": 319090871,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMxOTA5MDg3MQ==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 11,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 4,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 15,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319090871/reactions"
            },
            "updated_at": "2017-07-31T14:51:54Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319090871",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/11025490?v=4",
                "events_url": "https://api.github.com/users/powelleric/events{/privacy}",
                "followers_url": "https://api.github.com/users/powelleric/followers",
                "following_url": "https://api.github.com/users/powelleric/following{/other_user}",
                "gists_url": "https://api.github.com/users/powelleric/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/powelleric",
                "id": 11025490,
                "login": "powelleric",
                "node_id": "MDQ6VXNlcjExMDI1NDkw",
                "organizations_url": "https://api.github.com/users/powelleric/orgs",
                "received_events_url": "https://api.github.com/users/powelleric/received_events",
                "repos_url": "https://api.github.com/users/powelleric/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/powelleric/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/powelleric/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/powelleric",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I can echo what @powelleric suggested - I set the absolute paths to the `g:python*_host_prog` variables and the error has disappeared.",
            "created_at": "2017-08-03T00:17:16Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-319833088",
            "id": 319833088,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMxOTgzMzA4OA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 2,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 2,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319833088/reactions"
            },
            "updated_at": "2017-08-03T00:17:16Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319833088",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/2184696?v=4",
                "events_url": "https://api.github.com/users/mattrasband/events{/privacy}",
                "followers_url": "https://api.github.com/users/mattrasband/followers",
                "following_url": "https://api.github.com/users/mattrasband/following{/other_user}",
                "gists_url": "https://api.github.com/users/mattrasband/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/mattrasband",
                "id": 2184696,
                "login": "mattrasband",
                "node_id": "MDQ6VXNlcjIxODQ2OTY=",
                "organizations_url": "https://api.github.com/users/mattrasband/orgs",
                "received_events_url": "https://api.github.com/users/mattrasband/received_events",
                "repos_url": "https://api.github.com/users/mattrasband/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/mattrasband/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/mattrasband/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/mattrasband",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "@powelleric @mrasband Thank you, guys! At first I was a little skeptical since I had both `g:python*_host_prog` variables set. Turns out homebrew updated the python binaries and the absolute path for `python2` changed. Updated my settings file and the error is gone!\r\n\r\n@justinmk Feel free to close this as, for me, it's solved :)",
            "created_at": "2017-08-03T12:50:26Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-319959645",
            "id": 319959645,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMxOTk1OTY0NQ==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 1,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 1,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319959645/reactions"
            },
            "updated_at": "2017-08-03T12:50:26Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319959645",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/48519?v=4",
                "events_url": "https://api.github.com/users/filipekiss/events{/privacy}",
                "followers_url": "https://api.github.com/users/filipekiss/followers",
                "following_url": "https://api.github.com/users/filipekiss/following{/other_user}",
                "gists_url": "https://api.github.com/users/filipekiss/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/filipekiss",
                "id": 48519,
                "login": "filipekiss",
                "node_id": "MDQ6VXNlcjQ4NTE5",
                "organizations_url": "https://api.github.com/users/filipekiss/orgs",
                "received_events_url": "https://api.github.com/users/filipekiss/received_events",
                "repos_url": "https://api.github.com/users/filipekiss/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/filipekiss/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/filipekiss/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/filipekiss",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "Just came across Homebrew/homebrew-core#15746. This update to the formula seems to be what is causing `python` to break around some systems, including the reported error on neovim.",
            "created_at": "2017-08-03T13:27:02Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-319968404",
            "id": 319968404,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMxOTk2ODQwNA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319968404/reactions"
            },
            "updated_at": "2017-08-03T13:27:02Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/319968404",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/48519?v=4",
                "events_url": "https://api.github.com/users/filipekiss/events{/privacy}",
                "followers_url": "https://api.github.com/users/filipekiss/followers",
                "following_url": "https://api.github.com/users/filipekiss/following{/other_user}",
                "gists_url": "https://api.github.com/users/filipekiss/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/filipekiss",
                "id": 48519,
                "login": "filipekiss",
                "node_id": "MDQ6VXNlcjQ4NTE5",
                "organizations_url": "https://api.github.com/users/filipekiss/orgs",
                "received_events_url": "https://api.github.com/users/filipekiss/received_events",
                "repos_url": "https://api.github.com/users/filipekiss/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/filipekiss/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/filipekiss/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/filipekiss",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "There seems to be two issues here:\r\n\r\n* `provider#pythonx#Require()` doesn't check if `Detect` returned an error, and tries to start the empty string or something\r\n* even if `python[3]#Prog()` contains a non-empty string, we should still check the return value of `jobstart()` (if the user specified an invalid `g:python_host_prog` )",
            "created_at": "2017-08-05T18:16:08Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-320461298",
            "id": 320461298,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMyMDQ2MTI5OA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320461298/reactions"
            },
            "updated_at": "2017-08-05T18:16:08Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320461298",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1363104?v=4",
                "events_url": "https://api.github.com/users/bfredl/events{/privacy}",
                "followers_url": "https://api.github.com/users/bfredl/followers",
                "following_url": "https://api.github.com/users/bfredl/following{/other_user}",
                "gists_url": "https://api.github.com/users/bfredl/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/bfredl",
                "id": 1363104,
                "login": "bfredl",
                "node_id": "MDQ6VXNlcjEzNjMxMDQ=",
                "organizations_url": "https://api.github.com/users/bfredl/orgs",
                "received_events_url": "https://api.github.com/users/bfredl/received_events",
                "repos_url": "https://api.github.com/users/bfredl/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/bfredl/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/bfredl/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/bfredl",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Doesn't the `abort` attribute on `provider#pythonx#Require()` cause a failed `jobstart()` to raise an exception?  We shouldn't be getting to the `rpcrequest()` call if `jobstart()` failed.",
            "created_at": "2017-08-05T18:47:42Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-320463196",
            "id": 320463196,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMyMDQ2MzE5Ng==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320463196/reactions"
            },
            "updated_at": "2017-08-05T18:47:42Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320463196",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/112971?v=4",
                "events_url": "https://api.github.com/users/jamessan/events{/privacy}",
                "followers_url": "https://api.github.com/users/jamessan/followers",
                "following_url": "https://api.github.com/users/jamessan/following{/other_user}",
                "gists_url": "https://api.github.com/users/jamessan/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/jamessan",
                "id": 112971,
                "login": "jamessan",
                "node_id": "MDQ6VXNlcjExMjk3MQ==",
                "organizations_url": "https://api.github.com/users/jamessan/orgs",
                "received_events_url": "https://api.github.com/users/jamessan/received_events",
                "repos_url": "https://api.github.com/users/jamessan/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/jamessan/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/jamessan/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/jamessan",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "channel id can only be negative if `jobstart()` failed. But it looks like it also calls `EMSG` then...",
            "created_at": "2017-08-05T19:03:20Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-320464045",
            "id": 320464045,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMyMDQ2NDA0NQ==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320464045/reactions"
            },
            "updated_at": "2017-08-05T19:03:20Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/320464045",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1363104?v=4",
                "events_url": "https://api.github.com/users/bfredl/events{/privacy}",
                "followers_url": "https://api.github.com/users/bfredl/followers",
                "following_url": "https://api.github.com/users/bfredl/following{/other_user}",
                "gists_url": "https://api.github.com/users/bfredl/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/bfredl",
                "id": 1363104,
                "login": "bfredl",
                "node_id": "MDQ6VXNlcjEzNjMxMDQ=",
                "organizations_url": "https://api.github.com/users/bfredl/orgs",
                "received_events_url": "https://api.github.com/users/bfredl/received_events",
                "repos_url": "https://api.github.com/users/bfredl/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/bfredl/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/bfredl/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/bfredl",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I had a similar issue to @filipekiss, this is how I found/fixed it:\r\n\r\nFirst, I checked `which python`. It showed `/usr/bin/python`. That doesn't make sense as brew should be symlinking all installed applications into `/usr/local/bin`.\r\n\r\nSo I did an `ls -l /usr/local/bin | grep py` and saw that there is a `/usr/local/bin/python2` but no `/usr/local/bin/python`.\r\n\r\nSo I did `ln -s /usr/local/bin/python2 /usr/local/bin/python` to create the missing symlink. I checked python by doing `python --version` and everything was back to the brew setup. My errors in `nvim` were gone.\r\n\r\nI should also state that I have `g:python*_host_prog` set for both python versions to the `/usr/local/bin/` paths",
            "created_at": "2017-08-08T16:32:08Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-321010332",
            "id": 321010332,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMyMTAxMDMzMg==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 2,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 2,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/321010332/reactions"
            },
            "updated_at": "2017-08-08T16:36:07Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/321010332",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1425304?v=4",
                "events_url": "https://api.github.com/users/james2doyle/events{/privacy}",
                "followers_url": "https://api.github.com/users/james2doyle/followers",
                "following_url": "https://api.github.com/users/james2doyle/following{/other_user}",
                "gists_url": "https://api.github.com/users/james2doyle/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/james2doyle",
                "id": 1425304,
                "login": "james2doyle",
                "node_id": "MDQ6VXNlcjE0MjUzMDQ=",
                "organizations_url": "https://api.github.com/users/james2doyle/orgs",
                "received_events_url": "https://api.github.com/users/james2doyle/received_events",
                "repos_url": "https://api.github.com/users/james2doyle/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/james2doyle/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/james2doyle/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/james2doyle",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "Just ran into the issue but in my case I only had the python2 client. Running `pip3 install --user neovim` fixed the problem.",
            "created_at": "2017-10-22T06:27:34Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-338455133",
            "id": 338455133,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDMzODQ1NTEzMw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 4,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 1,
                "total_count": 5,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/338455133/reactions"
            },
            "updated_at": "2017-10-22T06:27:34Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/338455133",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1115287?v=4",
                "events_url": "https://api.github.com/users/DavidGamba/events{/privacy}",
                "followers_url": "https://api.github.com/users/DavidGamba/followers",
                "following_url": "https://api.github.com/users/DavidGamba/following{/other_user}",
                "gists_url": "https://api.github.com/users/DavidGamba/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/DavidGamba",
                "id": 1115287,
                "login": "DavidGamba",
                "node_id": "MDQ6VXNlcjExMTUyODc=",
                "organizations_url": "https://api.github.com/users/DavidGamba/orgs",
                "received_events_url": "https://api.github.com/users/DavidGamba/received_events",
                "repos_url": "https://api.github.com/users/DavidGamba/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/DavidGamba/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/DavidGamba/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/DavidGamba",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "NONE",
            "body": "I encounter the same issue. I fixed it by execute `:UpdateRemotePlugins` command. It seems like some old plugins break the python host detection process.",
            "created_at": "2018-03-19T05:13:11Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-374103224",
            "id": 374103224,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDM3NDEwMzIyNA==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 2,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 2,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/374103224/reactions"
            },
            "updated_at": "2018-03-19T05:28:55Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/374103224",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3964947?v=4",
                "events_url": "https://api.github.com/users/VMatrix1900/events{/privacy}",
                "followers_url": "https://api.github.com/users/VMatrix1900/followers",
                "following_url": "https://api.github.com/users/VMatrix1900/following{/other_user}",
                "gists_url": "https://api.github.com/users/VMatrix1900/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/VMatrix1900",
                "id": 3964947,
                "login": "VMatrix1900",
                "node_id": "MDQ6VXNlcjM5NjQ5NDc=",
                "organizations_url": "https://api.github.com/users/VMatrix1900/orgs",
                "received_events_url": "https://api.github.com/users/VMatrix1900/received_events",
                "repos_url": "https://api.github.com/users/VMatrix1900/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/VMatrix1900/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/VMatrix1900/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/VMatrix1900",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "CONTRIBUTOR",
            "body": "> It seems like that the nvim python provider detection is broken.\r\n\r\nThe detection is not perfect.\r\nI think it is not broken.\r\nYou should check the detection result by `:checkhealth`.",
            "created_at": "2018-03-19T05:29:28Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-374105293",
            "id": 374105293,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDM3NDEwNTI5Mw==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/374105293/reactions"
            },
            "updated_at": "2018-03-19T05:29:28Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/374105293",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/41495?v=4",
                "events_url": "https://api.github.com/users/Shougo/events{/privacy}",
                "followers_url": "https://api.github.com/users/Shougo/followers",
                "following_url": "https://api.github.com/users/Shougo/following{/other_user}",
                "gists_url": "https://api.github.com/users/Shougo/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/Shougo",
                "id": 41495,
                "login": "Shougo",
                "node_id": "MDQ6VXNlcjQxNDk1",
                "organizations_url": "https://api.github.com/users/Shougo/orgs",
                "received_events_url": "https://api.github.com/users/Shougo/received_events",
                "repos_url": "https://api.github.com/users/Shougo/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/Shougo/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/Shougo/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/Shougo",
                "user_view_type": "public"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "# Plan\r\n\r\nhttps://github.com/neovim/neovim/issues/7078#issuecomment-320461298",
            "created_at": "2018-09-25T09:31:16Z",
            "html_url": "https://github.com/neovim/neovim/issues/7078#issuecomment-424272051",
            "id": 424272051,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/7078",
            "node_id": "MDEyOklzc3VlQ29tbWVudDQyNDI3MjA1MQ==",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/424272051/reactions"
            },
            "updated_at": "2018-09-25T09:31:16Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/424272051",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/1359421?v=4",
                "events_url": "https://api.github.com/users/justinmk/events{/privacy}",
                "followers_url": "https://api.github.com/users/justinmk/followers",
                "following_url": "https://api.github.com/users/justinmk/following{/other_user}",
                "gists_url": "https://api.github.com/users/justinmk/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/justinmk",
                "id": 1359421,
                "login": "justinmk",
                "node_id": "MDQ6VXNlcjEzNTk0MjE=",
                "organizations_url": "https://api.github.com/users/justinmk/orgs",
                "received_events_url": "https://api.github.com/users/justinmk/received_events",
                "repos_url": "https://api.github.com/users/justinmk/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/justinmk/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/justinmk/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/justinmk",
                "user_view_type": "public"
            }
        }
    ],
    "comments": 13,
    "comments_url": "https://api.github.com/repos/neovim/neovim/issues/7078/comments",
    "created_at": "2017-07-26T12:30:28Z",
    "events_url": "https://api.github.com/repos/neovim/neovim/issues/7078/events",
    "html_url": "https://github.com/neovim/neovim/issues/7078",
    "id": 245706273,
    "labels": [
        {
            "color": "c5def5",
            "default": false,
            "description": "libnvim, Nvim RPC API",
            "id": 103819671,
            "name": "api",
            "node_id": "MDU6TGFiZWwxMDM4MTk2NzE=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/api"
        },
        {
            "color": "c5def5",
            "default": false,
            "description": null,
            "id": 144672485,
            "name": "provider",
            "node_id": "MDU6TGFiZWwxNDQ2NzI0ODU=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/provider"
        },
        {
            "color": "c5def5",
            "default": false,
            "description": "OS processes, spawn",
            "id": 182884815,
            "name": "job-control",
            "node_id": "MDU6TGFiZWwxODI4ODQ4MTU=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/job-control"
        },
        {
            "color": "c7def8",
            "default": false,
            "description": "channels, RPC, msgpack",
            "id": 242522707,
            "name": "channels-rpc",
            "node_id": "MDU6TGFiZWwyNDI1MjI3MDc=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/channels-rpc"
        },
        {
            "color": "0E8A16",
            "default": false,
            "description": "",
            "id": 606691254,
            "name": "has:plan",
            "node_id": "MDU6TGFiZWw2MDY2OTEyNTQ=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/has:plan"
        },
        {
            "color": "c5def5",
            "default": false,
            "description": "plugins as RPC coprocesses (node.js, python, etc)",
            "id": 652698463,
            "name": "remote-plugin",
            "node_id": "MDU6TGFiZWw2NTI2OTg0NjM=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/remote-plugin"
        }
    ],
    "labels_url": "https://api.github.com/repos/neovim/neovim/issues/7078/labels{/name}",
    "locked": false,
    "milestone": {
        "closed_at": null,
        "closed_issues": 706,
        "created_at": "2014-05-10T20:43:04Z",
        "creator": {
            "avatar_url": "https://avatars.githubusercontent.com/u/1359421?v=4",
            "events_url": "https://api.github.com/users/justinmk/events{/privacy}",
            "followers_url": "https://api.github.com/users/justinmk/followers",
            "following_url": "https://api.github.com/users/justinmk/following{/other_user}",
            "gists_url": "https://api.github.com/users/justinmk/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/justinmk",
            "id": 1359421,
            "login": "justinmk",
            "node_id": "MDQ6VXNlcjEzNTk0MjE=",
            "organizations_url": "https://api.github.com/users/justinmk/orgs",
            "received_events_url": "https://api.github.com/users/justinmk/received_events",
            "repos_url": "https://api.github.com/users/justinmk/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/justinmk/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/justinmk/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/justinmk",
            "user_view_type": "public"
        },
        "description": "Low priority items. We plan to work on this but don't have a target date.",
        "due_on": null,
        "html_url": "https://github.com/neovim/neovim/milestone/6",
        "id": 655037,
        "labels_url": "https://api.github.com/repos/neovim/neovim/milestones/6/labels",
        "node_id": "MDk6TWlsZXN0b25lNjU1MDM3",
        "number": 6,
        "open_issues": 636,
        "state": "open",
        "title": "backlog",
        "updated_at": "2025-03-17T22:41:08Z",
        "url": "https://api.github.com/repos/neovim/neovim/milestones/6"
    },
    "node_id": "MDU6SXNzdWUyNDU3MDYyNzM=",
    "number": 7078,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 12,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 12,
        "url": "https://api.github.com/repos/neovim/neovim/issues/7078/reactions"
    },
    "repository_url": "https://api.github.com/repos/neovim/neovim",
    "state": "open",
    "state_reason": null,
    "sub_issues_summary": {
        "completed": 0,
        "percent_completed": 0,
        "total": 0
    },
    "timeline_url": "https://api.github.com/repos/neovim/neovim/issues/7078/timeline",
    "title": "Error with Python Provider: Invalid argument: Channel id must be a positive integer",
    "updated_at": "2025-03-17T13:43:46Z",
    "url": "https://api.github.com/repos/neovim/neovim/issues/7078",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/48519?v=4",
        "events_url": "https://api.github.com/users/filipekiss/events{/privacy}",
        "followers_url": "https://api.github.com/users/filipekiss/followers",
        "following_url": "https://api.github.com/users/filipekiss/following{/other_user}",
        "gists_url": "https://api.github.com/users/filipekiss/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/filipekiss",
        "id": 48519,
        "login": "filipekiss",
        "node_id": "MDQ6VXNlcjQ4NTE5",
        "organizations_url": "https://api.github.com/users/filipekiss/orgs",
        "received_events_url": "https://api.github.com/users/filipekiss/received_events",
        "repos_url": "https://api.github.com/users/filipekiss/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/filipekiss/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/filipekiss/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/filipekiss",
        "user_view_type": "public"
    }
}